src/pages/workspaces/workspace/ShareWorkspaceModal/Collaborator.test.ts

```diff
@@ -48,6 +48,7 @@ describe('a Collaborator component', () => {
       createdBy: '',
       googleProject: 'test-project',
       bucketName: 'test-bucket',
+      lastModified: '',
```

Comment:
This test had empty strings for `createdBy`, so followed same pattern for `lastModified`.